server/middleware/validation.js

```diff
@@ -16,7 +17,10 @@ const validationErrors = {
     isValidWithinDate: label => `${label} must be within the last ${VALID_DAYS_RANGE} days`,
     validCaseReference: () => 'Case reference is not valid',
     contributionsFulfilled: () => 'Case contributions have to be completed or cancelled',
-    oneOf: () => 'Options are not valid'
+    oneOf: () => 'Options are not valid',
+    isValidMonth: label =>  `${label} must contain a real month`,
+    isYearWithinRange: label =>   `${label} must contain a valid year`,
```

Comment:
Don't use "valid".

Reply:
Updated to "Select at least one option", this is currently only used for when the user selects a complaints type and the message is overridden with "Select at least one complaint type option".